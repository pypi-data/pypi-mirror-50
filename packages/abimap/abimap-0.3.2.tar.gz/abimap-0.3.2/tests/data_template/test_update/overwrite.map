# This map file was created with PROGRAM_NAME_VERSION

LIBTC4_1_0_0
{
    global:
        another_symbol;
        symbol;
    local:
        *;
} ;

