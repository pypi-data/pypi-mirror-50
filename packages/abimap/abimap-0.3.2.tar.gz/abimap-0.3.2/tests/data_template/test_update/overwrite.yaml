# Testing overwriting the map
-
  input:
    args:
      - "update"
      - "--allow-abi-break"
      - "--out"
      - "overwrite.map"
      - "overwrite.map"
    stdin: "overwrite.in"
  output:
    file: "overwrite.out"
    stdout: "overwrite.stdout"
    warnings:
      - "Overwriting existing file 'overwrite.map'"
      - "Given paths in '--out' and 'file' are the same."
      - "Moving 'overwrite.map' to 'overwrite.map.old'."
      - "ABI break detected: symbols were removed."
    errors:
    exceptions:
