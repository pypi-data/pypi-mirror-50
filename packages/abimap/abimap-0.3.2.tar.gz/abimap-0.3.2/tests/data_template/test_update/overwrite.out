# This map file was updated with PROGRAM_NAME_VERSION

LIBTC4_2_0_0
{
    global:
        new_symbol;
        symbol;
    local:
        *;
} ;

